README.md

```diff
@@ -15,3 +15,10 @@ so we want to create a list that can help you query the companies much easier. b
 - [ING](https://www.ing.com/web/show)
 - [StyleShoots](https://styleshoots.com/)
 - [AIHR](https://www.aihr.com/homepage/)
+- [Zivver](https://workat.zivver.eu)
+- [Coolblue](https://www.careersatcoolblue.com/relocation/)
+- [Vanmoof](https://www.vanmoof.com/careers)
```

Comment:
I only found Iranian guys at the first 2 companies on your list.
do you know any Iranian guy who works at Vanmoof, Polarsteps, Convious, STRM or Silverflow?

Reply:
The question is, do you want to find companies that support Visa sponsorship or companies which hired Iranian people?! if the answer is the first one, it doesn't matter!